in_toto/in_toto_record.py

```diff
@@ -242,7 +252,8 @@ def main():
     # case the key is encrypted. Something that should not happen in the lib.
     key = None
     if args.key:
-      key = in_toto.util.prompt_import_rsa_key_from_file(args.key)
+      key = in_toto.util.prompt_import_private_key_from_file(args.key,
+                                                             args.key_type)
```

Comment:
Make sure that args.key_type is indented with 4 spaces.  See our guideline on wrapping long lines.

Reply:
Forgot to include a link: https://github.com/secure-systems-lab/code-style-guidelines#maximum-line-length